SwiftShieldExample/SwiftShieldSDK/SwiftShieldSDK/ContactsList.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ContactsList.swift
+//  SwiftShieldSDK
+//
+//  Created by Weidian on 22/12/19.
+//  Copyright © 2019 rockbruno. All rights reserved.
+//
+
+import Foundation
+
+
+public class ContactsList {
+    var contacts = Array<FzwubuMptrwvhVEwXAqMfTkHoQjlcQYY>()
```

Comment:
In the example, we should commit unobfuscated files

Reply:
Sorry, I didn't check, I will correct it.